index.html

```diff
@@ -1,14 +1,46 @@
 <!DOCTYPE html>
 <html lang="en">
+
 <head>
     <meta charset="UTF-8">
     <meta http-equiv="X-UA-Compatible" content="IE=edge">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <title>Hello Microverse</title>
+    <title>Temesgen Portfolio</title>
     <link rel="stylesheet" href="style.css">
 
 </head>
+
 <body>
-    <h1 class="main-header">Hello Microverse!</h1>
+    <header>
+        <h4>My Logo</h4>
+        <nav>
+            <a href="">
+            </a>
+            <img src="./img/menu.png" alt="web menu graphics">
```

Comment:
Please correct your image by giving it a height and width and  correct its place too
<img width="99" alt="image" src="https://user-images.githubusercontent.com/7243132/170825479-ca0c7a4f-1f47-4551-a78f-1ba29c5c5391.png">

you can do that : 
<img width="89" alt="image" src="https://user-images.githubusercontent.com/7243132/170825453-c20df3ac-588f-496a-a457-381d460c9727.png">